design/dedicated-osd-pod.md

```diff
@@ -36,12 +36,11 @@ We propose the following change to address the limitations.
 ### Create new OSDs
 | Sequence |Rook Operator | Device Discovery  | Device Provisioner   | Ceph OSD Deployment | 
 |---|---|---|---|---|
-| 1  | Read devices from cluster CRD and create an OSD Discovery deployment on each storage node  |   |   |
-| 2  | Watch raw device Configmap  | Discover unformatted devices and update raw device Configmap with the device paths and device information (size, type, etc)| | |
-| 3  | Detect raw device Configmap change, parse Configmap, extract device paths with matching filters defined in CRD, and create an Device Provisioner deployment for each device  || | |
-| 4  | Watch device provisioning Configmap | | Prepare OSDs, Persist OSD ID, datapath, and node info in a per node device provisioning Configmap | |
-| 5  | Detect device provisioning Configmap change, parse Configmap, extract OSD info, construct OSD Pod command and arg | | | |
-| 6  | Create one deployment per OSD | | |Start `ceph-osd` Daemon one Pod per device |
+| 1  | Read devices and node filters from cluster CRD |   |   |
```

Comment:
DId you intend to have step 1 be the device discovery? That column is blank now.